packages/ui/src/components/Catalog/DataListItem.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  DataListCell,
+  DataListItem,
+  DataListItemCells,
+  DataListItemRow,
+  Grid,
+  GridItem,
+  Label,
+} from '@patternfly/react-core';
+import { FunctionComponent } from 'react';
+import { IconResolver } from '../IconResolver';
+import { ITile } from './Catalog.models';
+import './DataListItem.scss';
+import { getTagColor } from './tag-color-resolver';
+
+interface ICatalogDataListItemProps {
+  tile: ITile;
+}
+
+export const CatalogDataListItem: FunctionComponent<ICatalogDataListItemProps> = (props) => {
+  return (
+    <DataListItem
+      aria-labelledby={props.tile.name}
+      key={props.tile.name}
+      id={props.tile.name}
+      className="catalog-data-list-item"
+    >
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell key="List item">
+              <Grid>
+                <GridItem
+                  sm={12}
+                  md={6}
+                  order={{
+                    default: '1',
```

Comment:
Could we please extract this definition into a dedicated object into another file, to avoid regenerating this object on every render? :smiley: